### ISSUE TITLE
merge

### ISSUE THREAD

**Issue by alias-noa** [chensvm/A-Dual-Stage-Attention-Based-Recurrent-Neural-Network-for-Time-Series-Prediction#4]
What version of tensorflow and keras were used with this?  Merge can no longer be imported and concatenate needs to be used but isn't working

**Comment by chuanchuan12138** [comment]
I solve it by install keras 2.1.6 rather than the newest version.

**Comment by cpoptic** [comment]
Yeah I encountered several errors when trying to port this code to work in Keras 2.2.4.

Making a new virtual environment with Keras 2.1 doesn't throw the deprecated keras.layers.Merge error.
